### PR TITLE
Figures 4 & 5: Scalings of Lineage Length and Lineage Number

### PR DESCRIPTION
**Pull request by namn12**

### CODE REVIEW

**Comment by shak360** [comment]
Merge when you're comfortable with it @namn12

File: lineage/figures/figure5.py [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
+from .Matplot_gen import Matplot_gen
+from .Fig_Gen import Lineages_per_Population_Figure
 
 
 def makeFigure():
     # Get list of axis objects
-    ax, f = getSetup((7, 6), (3, 4))
+    ax, f = getSetup((12, 3), (1, 3))
 
-    subplotLabel(ax[0], 'A')
```
Comment (aarmey) [inline lineage/figures/figure5.py]: The line removed here will place the subplot labels.

File: lineage/figures/figure4.py [hunk 2]
```diff
@@ -2,13 +2,17 @@
...
+from .Matplot_gen import Matplot_gen
+from .Fig_Gen import Lineage_Length
 
 
 def makeFigure():
     # Get list of axis objects
-    ax, f = getSetup((7, 6), (3, 4))
+    ax, f = getSetup((12, 3), (1, 3))
 
-    subplotLabel(ax[0], 'A')
```
Comment (aarmey) [inline lineage/figures/figure4.py]: Subplot labels.

**Comment by namn12** [comment]
> Merge when you're comfortable with it @namn12

it looks blocked from one of the ealirer commits

**Comment by namn12** [comment]
> > Merge when you're comfortable with it @namn12
> 
> it looks blocked from one of the ealirer commits @shak360

our old friend back again?

Traceback (most recent call last):
  File "./genFigures.py", line 14, in <module>
    ff = eval(nameOut + '.makeFigure()')
  File "<string>", line 1, in <module>
  File "/home/jenkins/jenkins/workspace/r-lab_lineage-growth_PR-109-head/lineage/figures/figure5.py", line 13, in makeFigure
    numb_of_lineage_h1, accuracy_h1, bern_MAS_h1, bern_2_h1, MASlocBern, locBern2, MASbeta, beta2, betaExp_MAS_h1, betaExp_2_h1 = Lineages_per_Population_Figure()
  File "/home/jenkins/jenkins/workspace/r-lab_lineage-growth_PR-109-head/lineage/figures/Fig_Gen.py", line 136, in Lineages_per_Population_Figure
    _, _, all_states, tHMMobj, _, _ = Analyze(X1, numStates)
  File "/home/jenkins/jenkins/workspace/r-lab_lineage-growth_PR-109-head/lineage/Analyze.py", line 22, in Analyze
    fit(tHMMobj, max_iter=200, verbose=True)
  File "/home/jenkins/jenkins/workspace/r-lab_lineage-growth_PR-109-head/lineage/BaumWelch.py", line 181, in fit
    get_nonleaf_NF_and_betas(tHMMobj, NF, betas)
  File "/home/jenkins/jenkins/workspace/r-lab_lineage-growth_PR-109-head/lineage/UpwardRecursion.py", line 177, in get_nonleaf_NF_and_betas
    assert np.allclose(betas_row_sum, 1.)
AssertionError

**Comment by namn12** [comment]
so the test that was failing was test_bw_4, which is a test for a 1 state lineage. all 1 state lineage models were failing because i initilzied E_bern to be 1/numStates. so when numStates was 1 (in this case), E_bern would start at 1. And the pdf of a cell that dies (ie bern =0) is zero in a distribuiton with bern=1, which was causing Betas to be NaN. So now the bern parameter is initialized randomly between (0.01 , 0.99) and the test passed and the AIC for one state seems to be working on the other branch. let me know if this sounds right to you guys @aarmey @shak360 and then can we merge this?

**Comment by aarmey** [comment]
Sounds reasonable to me.

**Comment by shak360** [comment]
@namn12 merge this and #110 . Good work and good catch on the bug.

**Pull request merged**